setup.cfg

```diff
@@ -4,7 +4,7 @@ max-line-length = 119
 multi_line_output = 3
 include_trailing_comma = True
 [mypy]
-python_version = 3.6
+python_version = 3.7
```

Comment:
Ahhhhhh, this is why mypy in CI was being so annoying about all the 3.7 stuff.

Can you remove the --python-version cli argument from mypy in CI? I'm pretty sure it won't be necessary anymore with this change.

Reply:
And I just wondered why it was working in CI and not on my machine :D

Reply:
I think we could move almost everything to `pyproject.toml`, now that we already have it, but flake8 doesn't seem to support it, so we won't be able to move to a single config file just yet.